package.json

```diff
@@ -420,9 +420,9 @@
     "semi": false
   },
   "dependencies": {
-    "@dcl/inspector": "^7.3.30-7064095632.commit-b118b46",
+    "@dcl/inspector": "^7.3.34",
```

Comment:
umm i didn't know we had the inspector as a dependency.. should we have it or just import the same version that is included in `@dcl/sdk`? I'm worried because of having these two separate we could end up with different versions at some point that do not work well together.